src/createScreen/createScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react';
+import './createScreen.less';
+import Button from '../components/button.tsx';
+import InsertName from './insertName.tsx';
+import RenderTypes from './renderTypes.tsx';
+import Stack from '@mui/material/Stack';
+
+export default function CreateScreen() {
+  const [playerName, setPlayerName] = useState('');
+
+  return (
+    <>
+      <h1>Yoncho Takeshi</h1>
```

Comment:
This h1 is basically the same why not just have it where the router is and render/change only the content below